test/unittests/libstorage/test_ConsensusPrecompiled.cpp

```diff
@@ -185,17 +185,17 @@ BOOST_AUTO_TEST_CASE(TestErrorNodeID)
     bytes out = consensusPrecompiled->call(context, bytesConstRef(&in));
     u256 count = 1;
     abi.abiOut(bytesConstRef(&out), count);
-    BOOST_TEST(count == 215u);
+    BOOST_TEST(count == 216u);
```

Comment:
Adding instructions in libstorage/Common.h maybe better.

Reply:
ok, add it.